user/src/main/java/com/chronica/user/logic/security/Jwt.java

```diff
@@ -0,0 +1,93 @@
+package com.chronica.user.logic.security;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+
+import java.io.File;
+import java.io.IOException;
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class Jwt {
```

Comment:
Czy ta klasa to JsonWebToken? Jeśli nie, to powinno się to nazywać jakoś w stylu JWTHandler